ceph/exporter.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 type versionedCollector interface {
-	Collect(chan<- prometheus.Metric, *Version)
+	Collect(chan<- prometheus.Metric, *Version, *sync.WaitGroup)
```

Comment:
I might have missed something, but I don't think there's any utility in passing the waitgroup down into the collector. I think we'd be better off wrapping the callsite with an anonymous function that calls `wg.Done()`. Then you wouldn't need that `localWg` dance in a few places.

Reply:
Yeah, agree. Pretty obvious in hindsight haha